tests/PKSim.Tests/Core/OriginDataMapperSpecs.cs

```diff
@@ -100,12 +100,19 @@ protected override Task Context()
 
    public class When_mapping_an_origin_data_to_snapshot_with_disease_state : concern_for_OriginDataMapper
    {
+      private OriginDataParameter _diseaseStateParameter;
+      private IDimension _timeDimension;
+
       protected override async Task Context()
       {
          await base.Context();
          _originData.DiseaseState = new DiseaseState {Name = "CKD"};
-         _originData.AddDiseaseStateParameter(new OriginDataParameter {Name = "Param", Value = 10, Unit = "mg"});
-         A.CallTo(() => _parameterMapper.ParameterFrom(10, "mg", A<IDimension>._)).Returns(new Parameter {Value = 10, Unit = "mg"});
+         _diseaseStateParameter = new OriginDataParameter {Name = "Param", Value = 60, Unit = "h"};
```

Comment:
Internal parameter are saved in base unit
Snapshot parameter are saved in display unit...